src/run_perf_test.py

```diff
@@ -36,6 +36,10 @@ def main():
     parser.add_argument("--stack", dest="stack", help="Stack name for performance test")
     parser.add_argument("--config", type=argparse.FileType("r"), help="Config file.", required=True)
     parser.add_argument("--keep", dest="keep", action="store_true", help="Do not delete the working temporary directory.")
+    parser.add_argument("--workload", default="nyc_taxis", help="Mensor (internal client) param - Workload name from OpenSeach Benchmark Workloads")
+    parser.add_argument("--workload-options", default="{}", help="Mensor (internal client) param - Json object with OpenSearch Benchmark arguments")
+    parser.add_argument("--warmup-iters", default=0, help="Mensor (internal client) param - Number of times to run a workload before collecting data")
+    parser.add_argument("--test-iters", default=1, help="Mensor (internal client) param - Number of times to run a workload")
```

Comment:
Can we have tests for the arguments as well?

Reply:
What sort of tests are you thinking? I've added the arguments to the run_perf_test test class, but since this file is really just passing the arguments around there's not much that I can think of to test here.

Reply:
I was thinking something like we did for https://github.com/opensearch-project/opensearch-build/blob/main/tests/test_run_sign.py.

Reply:
I see you have added the args already. Thanks for it.